website/docs/installing-podman-desktop/index.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 2
+title: Installing Podman Desktop
+description: You can install Podman Desktop on Windows, macOS, and Linux.
+tags: [podman-desktop, installing]
+keywords: [podman desktop, containers, podman, installing, installation]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing Podman Desktop
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows">
```

Comment:
Huge +1 on these tabs and defaulting to the current OS. It makes the nav so much more concise and you don't have to hurt/filter to current OS.